devices/MS5611/CalibrationData.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Iot.Device.MS5611
```

Comment:
About namespace: VS is getting crazy when I have the same name as a part of the namespace and a class name, I deliberately renamed that becasue of this. Weird that for Bmp180 it works fine, I compared projects file by file and I see no difference that would change this, any suggestion?

Reply:
I guess it's just a matter of using and full names. Works for others. Additional question: is there any other Ms56something so you may rename the namespace to take already in account there is a family behind? So that would kind of solve your issue even if you don't implement all of them right now.

Reply:
I didn't find any device that would be in the same family.